compute_tools/src/compute.rs

```diff
@@ -1463,6 +1533,16 @@ impl ComputeNode {
                         Ok(())
                     },
                 )?;
+
+                let postgresql_conf_path = pgdata_path.join("postgresql.conf");
+                if config::line_in_file(
+                    &postgresql_conf_path,
+                    "neon.disable_logical_replication_subscribers=false",
+                )? {
+                    info!("updated postgresql.conf to set neon.disable_logical_replication_subscribers=false");
+                } else {
+                    info!("postgresql.conf is up-to-date");
```

Comment:
Should we move it to after `pg_reload_conf()`? Because it's kinda up-to-date in all cases after that, having it only in else is a bit strange to me